operation/fhir-operation-term-cache/README.md

```diff
@@ -0,0 +1,7 @@
+The FHIR terminology subsystem assumes that artifacts with the same canonical URL and version are stable and thus can be stored in long-living cache space. If there is a need to iterate the content of a CodeSystem or ValueSet resource without updating the business version, additional operations are needed to clear the cache.
+
+If you are in development mode and always want to reload data, the simplest mechanism is to use the ``fhirServer/term/disableCaching``  configuration option. If you are in a hybrid or production mode and require both caching and dynamic content updates, the operations in this module can be used to clear specific cached resources.
```

Comment:
Not sure what putting two backticks around a word does, but we usually use just one:
```suggestion
If you are in development mode and always want to reload data, the simplest mechanism is to use the `fhirServer/term/disableCaching`  configuration option. If you are in a hybrid or production mode and require both caching and dynamic content updates, the operations in this module can be used to clear specific cached resources.
```